roles/content_view_promotion_rollback_publish/README.md

```diff
@@ -0,0 +1,86 @@
+theforeman.foreman.content_view_promotion_rollback_publish
+=========
+
+A role for automating the staging/promotion of Foreman Content-Views through various Organizations and LifeCycle Environemnts.  Can also be used for publishing new versions as well as rolling back.
+
+Requirements
+------------
+
+This role requires the theforeman.foreman module collection.
+
+Role Variables
+--------------
+
+The main amount of work is vars/main.yml.
+
+The primary dictionary is the organizations dictionary, which is formatted as such:
+```
+organizations:
+  org1:
+    lifecycle_environments:
+      - "Dev"
+      - "QA"
+      - "Prod"
+    content_views:
+      - "content-view1"
+      - "content-view2"
+  org2:
+    lifecycle_environments:
+      - "Dev2"
+      - "QA2"
+      - "Prod2"
+    content_views:
+      - "content-view3"
+      - "content-view4"
+```
+
+This can run against multiple organizations/lifecycle_environments/content-views or selected subsets.
+
+For example, if the previously mentioned dictionary describes *ALL* of my Foreman environment, but I only want to promote the 'Prod' lifecycle_environment in the content-view 'content-view1' in organization 'org', my dictionary would look like this:
+```
+organizations:
+  org1:
+    lifecycle_environments:
+      - "Prod"
+    content_views:
+      - "content-view1"
+```
+Items not described in the inventory will not be affected (the exception being Library when a new content_view_version needs to be published.
+
+If you only want to use this role for staging new content_view_versions, simply set the variable **publish_only** to **True** (this can be done in vars/main.yml or elsewhere).  This action will only publish a new content_view_version to the content_views described in the dictionary.
+
+If you want to rollback (reverse promote) lifecycle environments simply set the variable **rollback** to **True** (this can be done in vars/main.yml or elsewhere).  This action will only roll back lifecycle_environments in the content views described in main dictionary.
```

Comment:
```suggestion
If you want to rollback (reverse promote) lifecycle environments simply set the variable **rollback** to **True**.  This action will only roll back lifecycle_environments in the content views described in main dictionary.
```

I think this line could be dropped as users should not be editing the vars file inside the role. Same in the line above.